src/components/image/index.js

```diff
@@ -39,13 +41,19 @@ class Image extends PureBaseComponent {
     /**
      * Show image as a cover, full width, image (according to aspect ratio, default: 16:8)
      */
-    cover: PropTypes.bool
+    cover: PropTypes.bool,
+    /**
+     * the type of overly to place on top of the image
+     */
+    overlayType: PropTypes.oneOfType([PropTypes.oneOf(Object.keys(Overlay.overlayTypes)), PropTypes.string])
```

Comment:
you can just use the definition you have in Overlay component to avoid duplicity 
`overlayType: Overlay.propTypes.type`